rasterio/plot.py

```diff
@@ -72,21 +74,19 @@ def show(source, with_bounds=True, contour=False, contour_label_kws=None,
 
     """
     plt = get_plt()
-
     if isinstance(source, tuple):
         arr = source[0].read(source[1])
-
+        print(1111)
```

Comment:
👀

Reply:
sorry, this is for the code debug, we have deleted it.